src/foam/u2/view/CardSelectView.js

```diff
@@ -43,6 +43,15 @@ foam.CLASS({
     ^innerFlexer {
       min-width: -webkit-fill-available;
     }
+    .foam-u2-borders-CardBorder {
```

Comment:
Should be scoped, otherwise it will still ge global.

Reply:
:(